Gems/AWSCore/Code/Platform/Common/GetJsonSchemaPath_Common.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/IO/Path/Path.h>
+#include <AzCore/Utils/Utils.h>
+
+namespace AWSCore
+{
+    namespace Platform
+    {
+        AZ::IO::Path GetJsonSchemaPath()
+        {
+            static constexpr const char ResourceMapppingJsonSchemaFilePath[] =
+                "Gems/AWSCore/resource_mapping_schema.json";
+
+            AZ::IO::Path executablePath = AZ::IO::PathView(AZ::Utils::GetExecutableDirectory());
+            AZ::IO::Path jsonSchemaPath = (executablePath / ResourceMapppingJsonSchemaFilePath).LexicallyNormal();
+
+            return jsonSchemaPath;
```

Comment:
do we need to do a similar check on the final resolved path (else return ""), like in `GetJsonSchemaPath_Android.cpp` ?

Reply:
I don't think we need the check here since the output for AZ::JsonSerializationUtils::ReadJsonFile() will return an error and we've already verified it at https://github.com/o3de/o3de/blob/development/Gems/AWSCore/Code/Source/ResourceMapping/AWSResourceMappingManager.cpp#L250-L255